__tests__/errorsHandled.test.js

```diff
@@ -207,4 +206,61 @@ describe.only("GET /api/reviews/:review_id/", ()=>{
     })
 
 })
-})
\ No newline at end of file
+})
+
+
+
+
+    describe("GET /api/reviews", ()=>{
+        test("GET request returns an array of reviews with all properties of the reviews object PLUS  property of owner (which is the username from the users table) AND comment_count (which is the total count of all the comments with this article_id) ", ()=>{
+            return request(app)
+            .get('/api/reviews')
+            .expect(200)
+            .then((res)=>{
+                const {reviews} = res.body;
+                //console.log(reviews);
+                expect(reviews).toHaveLength(13);
+
+                       
+                reviews.forEach((review)=>{
+                    
+                    expect(review).toHaveProperty('owner');
+                    expect(review).toHaveProperty('title');
+                    expect(review).toHaveProperty('review_id');
+                    expect(review).toHaveProperty('category');
+                    expect(review).toHaveProperty('review_img_url');
+                    expect(review).toHaveProperty('review_id');
+                    expect(review).toHaveProperty('created_at');
+                    expect(review).toHaveProperty('votes');
+                    expect(review).toHaveProperty('review_body');
+                    expect(review).toHaveProperty
+                    ('designer');
+
+                    expect(review).toHaveProperty('comment_count');
```

Comment:
This is not thorough enough, we need to be testing that these things have the correct data type. We can do this `expect.objectContaining `